pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -52,6 +52,10 @@
     <UseApplicationTrust>false</UseApplicationTrust>
     <PublishWizardCompleted>true</PublishWizardCompleted>
     <BootstrapperEnabled>true</BootstrapperEnabled>
+    
+    <!-- Make the assembly name configurable -->
+    <AssemblyName Condition=" '$(MSBuildAssemblyName)' == '' ">Skyline-daily</AssemblyName>
+    <AssemblyName Condition=" '$(MSBuildAssemblyName)' != '' ">$(MSBuildAssemblyName)</AssemblyName>
   </PropertyGroup>

```

Comment:
What do you think about this change @brendanx67 ? There should be no change in the IDE, but it lets the command-line build override the default "Skyline-daily" assembly name. I used that ability to make --official override the name to "Skyline", which was necessary to fully test the admin installer's logic (because it changes the expected files based on the --official option).